facia-tool/public/js/widgets/message.js

```diff
@@ -0,0 +1,9 @@
+import ko from 'knockout';
+
+class Message {
+    constructor() {
+        this.frontsRemainingMessage = ko.observable();
```

Comment:
message is a generic class/widget, let's rename `frontsRemainingMessage` into some reusable, like `textMessage` or something like that